CHANGELOG.md

```diff
@@ -160,6 +160,7 @@ New deprecation(s):
 
 ### Other
 
+- **General**: Add command-line flag in Adapter and Operator to disable TLS in GRPC ([#5408](https://github.com/kedacore/keda/issues/5408))
```

Comment:
move this to Improvements section, as it is user facing change

Reply:
got it!